tests/cephfs/cephfs_utilsV1.py

```diff
@@ -444,16 +444,31 @@ def create_fs(self, client, vol_name, validate=True, **kwargs):
         Returns:
 
         """
-        fs_cmd = f"ceph fs create {vol_name}"
-        cmd_out, cmd_rc = client.exec_command(
-            sudo=True, cmd=fs_cmd, check_ec=kwargs.get("check_ec", True)
-        )
-        if validate:
-            out, rc = client.exec_command(sudo=True, cmd="ceph fs ls --format json")
-            volname_ls = json.loads(out)
-            if vol_name not in [i["name"] for i in volname_ls]:
-                raise CommandFailed(f"Creation of filesystem: {vol_name} failed")
-        return cmd_out, cmd_rc
+        out, err = client.exec_command(sudo=True, cmd="ceph version")
+        output = out.read().decode()
```

Comment:
Can you remove read().decode() here also

Reply:
@AmarnatReddy I have remove in the line as well